standby/src/lib.rs

```diff
@@ -355,43 +380,54 @@ impl Standby {
 
                 bystanders.is_empty()
             }
-            None => return,
+            None => {
+                log::trace!("Message {} has no reaction bystanders", message_id);
+
+                return;
+            }
         };
 
         if remove {
+            log::trace!("Removing message {}", message_id);
             reactions.remove(&message_id);
         }
     }
 
     /// Iterate over bystanders and remove the ones that match the predicate.
     fn iter_bystanders<E: Clone>(&self, bystanders: &mut Vec<Bystander<E>>, event: &E) {
+        log::trace!("Iterating over bystanders: {:?}", bystanders);
+
         let mut idx = 0;
 
         while idx < bystanders.len() {
+            log::trace!("Checking bystander");
             let bystander = &mut bystanders[idx];
 
             let sender = match bystander.sender.take() {
                 Some(sender) => sender,
                 None => {
-                    idx += 1;
+                    log::trace!("Bystander has no sender, removing");
```

Comment:
What's the reasoning for removing the `idx +=1` here?

Reply:
Oops that was meant to only be in the other PR, I'll remove that, thanks